### PR TITLE
Tooltip and mobile layout change

### PR DESCRIPTION
**Pull request by alexunique0519**
### Description
The tooltip icon is replaced with a more information link. The layout dimensions have been changed for mobile view.

List of proposed changes:
- Tooltips icon is replaced with a link
- Extended the main content width for mobile views
- Fixed the "what you told us" column issue

### CODE REVIEW

File: components/Forms/Radio.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const Radio: React.VFC<InputProps> = observer((props) => {
 
   return (
     <div className="radio">
-      <div className="mb-2.5">
+      <div className="">
```
Comment (habibaz) [inline components/Forms/Radio.tsx]: do we need to keep this div

File: components/Forms/Radio.tsx [hunk 2]
```diff
@@ -45,17 +45,21 @@ export const Radio: React.VFC<InputProps> = observer((props) => {
             dangerouslySetInnerHTML={{ __html: label }}
           ></span>
         </label>
-        <span>
+        {/* <span>
```
Comment (habibaz) [inline components/Forms/Radio.tsx]: if this is not needed can we please remove this

Comment (alexunique0519) [reply components/Forms/Radio.tsx]: Yes, I just reverted this change, I will create a separate PR to address that change

File: components/Forms/Radio.tsx (hunk 1 above)
Comment (alexunique0519) [reply components/Forms/Radio.tsx]: Yes, otherwise the "more information" link will come to the end of this line. I deleted the "className".

**Comment by alex-solo** [comment]
Screen Shot 2022-06-28 at 3 17 55 PM

I will approve this PR since it wasn't addressing the issue in the image above, but it's something to fix here or in a separate PR.

**Review by alex-solo — APPROVED** [review]
Please see comment I left

**Review by JeremyKennedy — CHANGES_REQUESTED** [review]
In general it would be better to split PRs into specific items, by grouping unrelated changes in, it's a bit harder to review/test. Now since I think we'll be going with a different tooltip/more-info design, the other changes in this PR will have to wait until the redesign is done. Though, we could pull out the unrelated changes and merge those in while the tooltip redesign is in progress... 🤔

**Comment by JeremyKennedy** [comment]
I've split this PR into two separate ones, #317 and #318. Together, those should be exactly the same as this one, so I will close this one.

**Pull request closed without merging**